policyengine_us/variables/gov/states/de/tax/income/credits/de_eitc.py

```diff
@@ -0,0 +1,17 @@
+from policyengine_us.model_api import *
+
+
+class de_eitc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Delaware EITC"
+    unit = USD
+    documentation = "Refundable and non-refundable Delaware EITC page 8"
```

Comment:
Instead add `#page=8` to the reference URL. Though I don't see it on page 8.
```suggestion
    documentation = "Refundable and non-refundable Delaware EITC"
```